### DISCUSSION TITLE
Week 1 and 2 - FAIR and CARE data principles, Open Science

### DISCUSSION THREAD

**Discussion by eculler** [earthlab-education/Earth-Analytics-2023-01-Intro#21] [Discussion submissions]
**NOTE** Most of the sections didn't get to discuss the *Data Feminism* reading last week, so we're extending this discussion. We've added one more short paper/comic about Open Access publication - another component of Open Science.

Readings for week 1 (Open data, FAIR, and CARE):

- [FAIR data principles](https://www.go-fair.org/fair-principles/) from GO FAIR. [GO FAIR. (n.d.). FAIR Principles. GO FAIR. Retrieved August 28, 2023, from https://www.go-fair.org/fair-principles/]
- [FAIR and CARE data principles](https://www.gida-global.org/care) from the Global Indigenous Data Alliance,  [CARE Principles. (2023, January 23). Global Indigenous Data Alliance. https://www.gida-global.org/care]
- [Data Feminism](https://data-feminism.mitpress.mit.edu/) This is the WHOLE BOOK - For class right now, just read the text on this front page so you understand where the authors are coming from. 
- [The Numbers Don't Speak for Themselves - Situating Data on the Wild Wild Web](https://canvas.colorado.edu/courses/95385/files/66930769) from _Data Feminism_ [D’Ignazio, C., & Klein, L. F. (2020). Data Feminism. MIT Press.]

Readings added for week 2:

- [Impacts, Adaptation, and Vulnerability Figure 9.5]( https://www.ipcc.ch/report/ar6/wg2/figures/chapter-9/figure-9-005) from the IPCC [Trisos, C.H., I.O. Adelekan, E. Totin, A. Ayanlade, J. Efitre, A. Gemeda, K. Kalaba, C. Lennard, C. Masao, Y. Mgaya, G. Ngaruiya, D. Olago, N.P. Simpson, and S. Zakieldeen, 2022: Africa. In: Climate Change 2022: Impacts, Adaptation and Vulnerability. Contribution of Working Group II to the Sixth Assessment Report of the Intergovernmental Panel on Climate Change [H.-O. Pörtner, D.C. Roberts, M. Tignor, E.S. Poloczanska, K. Mintenbeck, A. Alegría, M. Craig, S. Langsdorf, S. Löschke, V. Möller, A. Okem, B. Rama (eds.)]. Cambridge University Press, Cambridge, UK and New York, NY, USA, pp. 1285–1455, doi:10.1017/9781009325844.011.]
- [The Fair Open Access Principles](https://www.fairopenaccess.org/the-fair-open-access-principles/) from the Fair Open Access Alliance [Fair Open Access | The Fair Open Access principles. (n.d.). Retrieved September 6, 2023, from https://www.fairopenaccess.org/the-fair-open-access-principles/]
- [The Rise of Open Science](https://www.science.org/doi/epdf/10.1126/science.342.6154.58) from Science [Munroe, R., Kaser, J., & Malakoff, D. (2013). The Rise of Open Access. Science, 342(6154), 58–59. https://doi.org/10.1126/science.342.6154.58]

Some questions to get you started:

   1. Why is open data important?
   2. What are some ways that you as a data manager or scientist can make your data more open and reusable?
   3. What are some potential drawbacks of open data?
   4. What are some ways you can help ensure that the Earth Observation data you work with is placed in context?
   5. Why is open access publication important?
   6. What are some things you can do to make sure your work is accessible?

**Comment by eculler** [comment]
Taking some notes from @lijo8146 's section:

- What does 'Interoperable' mean? It means that regardless of the operating system (MacOS, Windows, Linux) or system (cloud, desktop, mobile) that data and code still works.
- How are Indigenous communities getting the CARE principles out there? It's evolving! But if you work with Indigenous communities it's important to take them into account. For example - the USGS takes a lot of great LiDAR data, but communities might not want LiDAR of their land published! And also remember that each Tribe is different and there's no one right way to interact with all.

**Comment by megan8617** [comment]
I just submitted a dataset to a data repository last week, and now I'm thinking I want to pull out the data from the format I submitted in and at least add a separate (really long) CSV file to make it easier for others to use (should they ever, ever want to)... I look forward to revisiting these readings with another cup of coffee and a little more time.

**Reply by alexiemillikin** [reply]
I agree! Publishing underlying data is not the norm in my field, and when it is done it's usually in Excel and with minimal information about data processing. As I'm working on processing a large dataset right now I'm hoping these readings with help me think about the best practices for making these data and my methods transparent.

**Reply by annieM459** [reply]
I agree! I've been very lucky to collaborate with scientists that provide very good metadata to help me understand what data I am working with. Although it's a pain, it works great for people handling the data without much instruction.

**Reply by eculler** [reply]
Great idea @megan8617! Depending on how long really long is, you might consider other strategies, like a database or several CSV files that can be concatenated or joined as needed. For example - if you have multiple sites and multiple data points at each site, you can put information that gets repeated for each data point in the site into a separate file to make the files smaller. Another common strategy to reduce file size is to have one file per site, or one file per year, etc.

**Comment by misterskye** [comment]
The Data Feminism is pretty exciting---Anything to counter the White/Male model!

**Comment by Klydascope** [comment]
The data feminism and CARE principles make me feel more welcome in this realm! I'm glad that these ideas are part of the spearhead of this program.

**Comment by alexiemillikin** [comment]
I'm very excited to read about Data Feminism! I think it's so important to recognize that data doesn't necessarily "speak for itself" and can be displayed/manipulated to support an agenda (in the most extreme cases) and is often interpreted without recognizing underlying biases.

**Reply by aholcomb23** [reply]
Well said! No data is truly "Raw Data" and ends up going through multiple "filters" before someone gets the chance to see it and interpret it for themselves. Whether there is intentional manipulation or accidental data skewing, data never speaks for itself.

**Reply by thku8507** [reply]
Well put! And the biases can start even before the data are collected--without any information about how the study was designed, what's included in the sample and who carried out the study, we lack the context of the "knowledge infrastructure" that made the dataset possible and that could influence our own interpretations of it.

**Reply by JulesMRuef** [reply]
Thinking into the future, I think it could be _really_  helpful for schools/universities to amend data science programs and add focuses on ethics, bias, and social responsibility! If we could teach the next generation to assess bias at each step and ask questions like “Is this ethically beneficial & analyzed?", I think we could curtail _some_ of our current problems.

**Comment by nquarder** [comment]
Notes from Discussion Section 2 (Weds 4-5:30pm) - Agencies like USDA have tons of data that largely goes unused; not a ton of demand outside of academia/research; working with private data and issues related to data ownership; how does CARE principles apply to Indigenous communities outside the United States?; @peterkobylarz brought up that we should continue to circle back to **FAIR** and **CARE** throughout the semester to continue the discussions. 

Didn't have enough time to discuss the Data Feminism piece in enough detail.

@JKinsler asked about some examples of folks who are working in the space around the CARE principles. I will continue looking through the literature but this recent article in Nature about applying the CARE principles in bio and ecology research (Jennings et al, 2023): [https://www.nature.com/articles/s41559-023-02161-2](https://www.nature.com/articles/s41559-023-02161-2)

Take a look at this and we can continue talking about these ideas as the semester goes on.

**Reply by eculler** [reply]
@JKinsler I'd recommend checking out the work of Frank Lake

**Comment by peterkobylarz** [comment]
I'm excited about Data Feminism as well and looking forward to reading more of that book.

I've definitely encountered "zombie data" before. It is so frustrating to find the dataset that you think you need but be smothered by an inability to sort out extraneous data from what you actually need, or difficulty to find context in poorly documented coded attribute fields, or just the lack of user friendly tools to download and parse subsets of data.

**Comment by Adriannaluger** [comment]
Open data is important because as we discussed in my section tax payer dollars are being used to make the data thus it should be available to the public. Additionally, I imagine having more open data for public use would increase findings in various fields and prevent people from potentially replicating data and be able to work off of each other more and be more productive. To make data more reusable you should follow the FAIR and CARE outlines to make it easier to reuse but also to make sure you are doing so ethically. Some drawbacks of open data that were discussed in data feminism are related to the idea that data is power. This means that is can be used for good but can also be used to discriminate, police, and surveil against certain people/groups. Which is why it is so important to use CARE and to think about whos interests are in mind during the making of the data. In Numbers Dont Speak for Themsleves they discussed how being able to tell if data is placed in the correct context can and is very difficult. The first step is always to read the metadata but sometimes this can be scarce, which results in zombie data. If this is the case then maybe it isnt the right data set for you? Or maybe doing some deep dive searching for the information may allow you to find the info you are looking for, I am a bit unsure of what is best to do in this situation?
I thought the data feminism piece was really interesting because I have yet been exposed to something that intersects "hard" science fields with "soft" science fields. I think it is really important to do so and am excited to continue to read more into the book and how connecting these ideas help to advance our data in technical and ethical ways.

**Reply by nbol48** [reply]
I think your comments on open data leading to increased scientific efficiency are spot on! Data collected for one purpose can often be applicable to another, particularly with an increasingly sophisticated suite of sensors recording multiple data streams simultaneously.

**Reply by annettebalmes** [reply]
I do agree that there are some ethical concerns when it comes to open data, I also think your point relays with accessibility barriers. Not everyone has equal access to open data due to technological, language, or digital literacy barriers. This can result in the exclusion of certain groups from the benefits of open data and that is another big concern.

**Comment by edchanonline** [comment]
I sometimes follow the work of Scott Reinhard at NYT (https://www.nytimes.com/by/scott-reinhard). His work wouldn't be possible because of open data and open science. I think about that a lot when I see good reportings on wild fires, military conflicts, etc. I was born in San Francisco so my recent favorite is this piece about [San Francisco Fog]( https://www.nytimes.com/interactive/2022/09/14/climate/san-francisco-fog.html). Another recent observation is that the industry is also catching up with the concept of open data. Umbra is publishing their large repository of SAR images at https://registry.opendata.aws/umbra-open-data/

Exciting time!

**Reply by eculler** [reply]
Cool! The world of accessing proprietary data can be deeply weird - like, it's available but only if you know someone from a particular country, or data for a particular time and place will just silently disappear for awhile. BUT a lot of revolutionary science has originally relied on proprietary or other non-open data, so it's not a simple issue even before you consider things like data sovereignty!

**Comment by jsmi374** [comment]
In recent years, there has been a significant push in the scientific community to make data/ datasets, algorithms, and even computer programs available to public end users. For example, some research journals require authors to include "supporting information" as an appendix to their publication(s). Supporting information often includes raw data, programs, methods, video files, animations, figures, etc. I have found this to be very useful when trying to reproduce results from a specific study, or incorporate methods from a publication into research programs or data analysis workflows. However, scientist can be withholding..... either inadvertently or on purpose. Some of the supporting information repositories I have come across and used are stripped down, lack context, or supply code that is undocumented or buggy. I think we are getting there as a community, and the effort is surely recognized, but the rules and protocols to adhere to seem to either, not recognized or known across the board. The mere existence of the FAIR and CARE Principles is exciting to me. It is sad that  I was unaware of these prior to enrolling in this class, but that is why I am here I suppose! 

Continuity is needed, and it should be prioritized. I fear that there is not currently a unified effort being made by organizations, agencies, or academics in respective fields to make this happen. 

Is there a specific sector/ field that is adhering to the FAIR and CARE principles well at the moment? 

or do some do it well and others not so well? 

Looking forward to meeting everyone, and learning this semester. I hope everyone has a wonderful weekend!

**Reply by eculler** [reply]
The CARE principles are very new, so folks are still figuring out the best practices. Right now I think the models are primarily Indigenous scientists that are working hard to follow these principles and get others to adopt them, like Frank Lake who I mentioned earlier. 

As far as FAIR principles - there's some great open data organizations out there! One example is the National Ecological Observatory Network, or NEON. We'll be using some of their data this semester for sure. The USGS and NASA have a ton of data that's available, although I think it could be easier to access. There's also citizen science organizations out there, like iNaturalist, that make their data very accessible.

**Comment by taylor-obrien** [comment]
I found the FAIR and CARE data principles to be extremely helpful and relevant to the work that I am conducting for my thesis. I am currently using locally observed events in Alaska, which includes many Indigenous community members. I will be keeping the FAIR and CARE data principles in my mind as I am conducting, developing, and sharing my data.

**Comment by Adriannaluger** [comment]
Week 2 comments: I really enjoyed the comic "The rise of open science" I liked how it gave some background history on open source data and showed how prices began to emerge in 2001 and have soon become what they are today. It's really interesting how they begin the comic by emphasizing the exponentially growing number of publications that are written each year and then show how financial restrictions are becoming more and more common/strict. It seems like even though the quantity of publications has grown enormously, the amount that is available does not match this growing statistic (I assume it has grown but not nearly parallel to the amount of total publications), which is really unfortunate.

**Comment by millie-spencer** [comment]
I really enjoyed D'Ignazio and Klein asking the questions "data by whom" and "data for whom" -- as a geographer, these questions felt akin to the conversations had in feminist and marxist geography that encourage an interrogation of the systems and legacies of our research. I think these questions add a layer of complexity to the conversation on data access and availability; on the one hand, increasing access to data improves agency over information and decision making, which can empower historically marginalized groups. On the other hand, there's the issue of data sovereignty and consent, and the potential for sharing data that's not one's own to do harm. On both sides, increasing representation improves the likelihood that data is gathered and disseminated in a more just manner.

**Comment by nbol48** [comment]
The differences in scientific research according to geographical area is one way that big data can have a profound effect on the community. As modeling tools become more efficient and can process larger data sets with greater accuracy, data scientists can continue to apply them on a more global scale, increasing the access to scientific research for historically understudied regions.

**Reply by eculler** [reply]
So true! Stay tuned - we're going to talk more about equity in ground truth data. Lots of satellite products have to be corrected to eliminate biases, and that can only be done in places with relatively dense ground measurements!

**Reply by peterkobylarz** [reply]
I definitely agree with this. I think it is also important to point out that big data needs to be able to broaden access to this data for scientist and researchers that actually live in the areas under-represented in figure 9.5 rather than just allowing scientists from Western countries to remotely conduct research on these regions.

**Reply by lucap1211** [reply]
I also agree that big data can widen the geographical areas that researchers can study, but I am a bit concerned if big data can have a comparable "global skill" when verification and other more resource-intensive processes are unevenly applied given differences in funding/access/etc...

**Reply by jsmi374** [reply]
From a community standpoint, I think it is also important to recognize the degree to which information (based on data) is generated, shared, and received by the general population, and how important it is ... for that transfer of knowledge to be be unbiased. It is sad, that in todays world, the "inconvenient truths" put out by scientist are often disregarded or attacked publicly. There is almost the need for  community assigned advocacy groups in regions impacted by various environmental factors. 

I am from South Louisiana where the effects of climate change stare the general population in the face. The level of community outreach and engagement related to the climatic issues has gotten better in recent years, but is pretty much still non-existent.

**Comment by peterkobylarz** [comment]
Open access earth observation platforms (such as ESA Sentinel-2) are really exciting to me as both extremely impressive engineering accomplishments and amazing sources of great data. Not only do they generate high quality data across the entire world, they allow scientists and researchers in regions that may be under represented as illustrated by figure 9.5 to gain access to this data without barriers that may have existed in the past. The fact that these platforms are operated and maintained by Western countries with troublesome imperial/colonial histories raises some challenging questions from the perspective of data feminism...

While private earth observations platforms like Planet Labs are extremely impressive and represent a huge opportunity for earth data science, in my experience it is very difficult to actually get your hands on their data. The data gatekeeping we see here feels like the antithesis of open access. I've heard anecdotally that their largest customers are the Department of Defense and private equity firms and I suspect that there are a lot of contracts involved that would preclude further access of data to other competitors/enemies. Commercialization of data is at the core of many omnipresent institutions in modern life and it is hard to imagine a private sector firm like Planet functioning in another way, but I hope there is a movement in the future for companies like this to make their data more available for environmental and resiliency efforts.

**Reply by eculler** [reply]
Definitely! I've been told that Planet data has student and academic allocations and also I think provides data for some grant-making agencies. But it's true that you can't just give someone else a link or some code to download it for themselves. I'm especially interested in some of the high-resolution thermal data that's coming out on private platforms, because we've had a lot of interest from urban areas looking to analyze urban heat island, and existing thermal sensors are far too low resolution. This kind of data is really important for equity and allocating funds where it is most needed. But as we discussed when talking about Data Sovereignty the very groups that could benefit could also be harmed and it's important to discuss data use with affected communities.

Defense is also a prime reason that many space agencies and others do not make their data totally open. This is especially true of real-time high resolution data, which can be used to identify and target individuals or military operations in other countries.

**Reply by JulesMRuef** [reply]
Small side note, I recently submitted a request to access Planet Labs as a student and stumbled upon this rather contradictory passage in their company goals/aims. One of their goals reads “We will operate the world's largest fleet of Earth imaging satellites to frequently image the entire planet and provide open access to that information.” They later go on to say, “The best way to tap into the full potential [of the Flock 1 data] is to make it _universally accessible with affordable pricing_. ” 

After spending two weeks chatting about open science this statement is pretty contradictory. Perhaps Planet Labs might consider clarifying that their data that are readily accessible to users, not that they are free of charge.

**Comment by ellenalamont17** [comment]
Fair and Open Access

This, in theory, is a fantastic idea. Providing wider audiences access to data and information means a more informed society, especially in the face of growing AI and machine learning applications. If this could be combined with the growing inclusion of "lay-person" abstracts/summaries, then communicating ideas and advances would become even more accessible. The problem with open access, however, is a systemic one. Historically, access to funding for research and publishing has been restricted to a very specific, non-diverse community workers often in only the more economically developed countries. This presents a sample bias in modern AI applications and a philosophical bias in ideas, procedures, concepts, etc. If, as indicated in the [The Rise of Open Science](https://www.science.org/doi/epdf/10.1126/science.342.6154.58) cartoon, these older texts are being digitized and funneled into the publication pool, perpetuating the historic problem. While more workers from diverse and historically under represented communities are beginning to participate in these literary endeavors, these works are still lost in the sea of existing literature. With time, this issue may start to balance out, but that may will be a distant future. Thus, when working with open access information, especially in an AI environment, we need to be careful to scrutinize result, patterns, and trends observed.

**Reply by eculler** [reply]
Great point! On the flip side, lack of open access means that many communities can't use research that could directly apply to local conservation, restoration, and adaptation efforts. It's really important to communicate science beyond academic journals (one reason that we focus on the 'blog post' genre in this class)

**Comment by edchanonline** [comment]
The XKCD comic provoked additional thinking into the cost of open access. When it comes to Open Data, I think of Landsat data on AWS a lot. This FAQ from USGS explains if there is any cost associated to accessing Landsat data via AWS. It seems fair you have to pay for any fees related to storage, processing, and egress, while the licensing the data is free. Does it really adhere to the FAIR data principles though? Perhaps by providing free access through Earth Explorer is good enough? Is this analogous to accessing academic papers? Who are paying the fees to host all these academic papers? Authors? Readers? Others? Or, is open access subsidized by other revenue streams? If subsidized, does it introduce biases?

**Reply by eculler** [reply]
Great point! For very large amounts of data, it can be more costly to pay to transfer it to another platform than to pay for computing on AWS.

There are a number of ways to come by computing resources. Many grant-making agencies provide computing and data access for grant recipients. The European Space Agency has such a facility that is available at different costs for different types of users. It's pretty easy to apply for computing time on JetStream2 (NSF) or Cyverse (also NSF), but I actually don't know if you can apply if you aren't from the US or working with someone who is. And of course some parts of the world wouldn't be able to access online computing facilities out of country.

It's a bit of a challenge, because these things are supposed to be easy to apply for, but if they are too easy then they tend to get attacked by Bitcoin miners...

**Comment by lucap1211** [comment]
The question "What are some ways you can help ensure that the Earth Observation data you work with is placed in context?" made me think how beyond data availability and quality, the amount of funding that supports the data-gathering process can also lead to significant inequalities. Now, the way we can use drones and other ways to collect data are resource-intensive processes and limit for which region the analyzed data is even available. In my opinion, researching, for example, the impacts of climate change is highly focused on North America and Central Europe while the adaptation capacity is disproportionally distributed globally. By uneven funding distribution, researchers might never realize the need to quantify climate change impacts and assist adaptation in regions where "quality data" is lacking. 

Furthermore, I found the Data Feminism book essential to better understand how we can't even neglect emotions when data visualization decisions are made. I never believed in the idea that "data can speak for itself", furthermore, as an athlete I always thought that "results speak for themselves" is also an extremely limited view. The process that leads to any result or the analysis that yields research findings shouldn't be interpreted without accounting for the context and space-time complex within which information is created/generated. 

@eculler

**Comment by thku8507** [comment]
It's interesting to think about how the movement towards open data and open access publication might give rise to unexpected and even counterproductive effects on the accessibility and openness of science. [The Biden Administration's recent executive order](https://www.insidehighered.com/news/2022/08/26/us-mandates-immediate-public-access-taxpayer-funded-research) mandating free and immediate access to government-funded research (with some limitations), eliminating a previous optional 12-month embargo period, is a big step towards improving the accessibility of scientific data. But by requiring that taxpayer-funded research be accessible without a paywall, this order will have varying impacts on different fields, as some institutions and fields have more funding available than others to cover the publisher fees required to remove a paywall. Though I think this new policy is a definite step forward, it's important to think about how it could shift power in terms of who has the ability and financial means to publish. A quote from that article ^^ to illustrate this concern: 

"David Burbach, an associate professor of national security affairs at the U.S. Naval War College, wrote on Twitter that he favors open access but expressed concern about equity for scholars across varied institutions.

'This would be a huge hit to people in institutions like mine, where generally we cannot get funding for access fees,' Burbach wrote. 'Perhaps with a govt wide policy requirement that would change.'"

**Comment by JulesMRuef** [comment]
I was chatting with some Earth Science folks today about how science (very generally) measures the worth of a published piece of research. The general consensus was that an article's or journal's value is measured by citations (articles cited more have a larger impact). 

By definition open access articles have greater visibility. Surprisingly, I also learned that even though most academic institutions pay for access to journals, open-access articles still have greater visibility and increased citation rates. In my eyes, this means that open science is essentially facilitating new collaborations, that science needs to be open access to facilitate this collaborative process. 

In any case, I had a lot of fun chatting with bright folks and thought I'd share some of the highlights!

**Comment by agneslh** [comment]
One thing that came up in our week 2 discussion was how expensive it is to actually pay for your work to get published in an open access journal. I personally didn't know how expensive it was and made me really think a lot about open access work in my field. One topic that immediately came to mind with emerging contaminant contamination is two prominent articles about fluoropolymers being of low concern for environmental and human health. Both of these papers are published by a group of scientists who are employed by huge polymer companies like Gore-Tex, and DuPont (meaning the papers will be a little biased). However, these papers are cited very frequently. This led me to thinking about how biased open access papers are, and then in turn how this can impact the rest of the field (biasing towards open source research = bias towards money and industry?).

Additionally, the 'how much science is there?' comic depicting the rate at which new papers are published made me think about citations. To me it seems like the rate at which new papers are published must impact how accurately sources are cited in papers. I've found that in my research there are a lot of 'circular sourcing' or that's what I call it, when papers kind of all cite (point) to each other without actually identifying what hard data the statement is from. It seems like maybe the old more traditional way of citing sources might quickly become antiquated - are there any new schools of thought on citing sources that account for the wealth of data available now/ rate of paper publication?

**Comment by annettebalmes** [comment]
I want to focus on the drawbacks of open science, as we clearly all understand the benefits.

Open data means sharing information with everyone for different uses. It's great, but it also has some problems. People's privacy can be at risk if personal or sensitive data is not properly protected. Sometimes, the data isn't accurate or complete, which can lead to bad decisions. Open data might also keep existing biases, making decisions unfair. There's a danger of hackers getting sensitive info, and there's so much data that it can be overwhelming. Who owns the data and how it can be used can be confusing. Keeping open data going takes a lot of resources. Some people can't easily access the data, and it might be misused for bad things. Ethical problems can come up, especially with sensitive data. Using open data can also lead to unexpected problems, like changing neighborhoods. And following the rules when using open data can be tricky. Still, we can make open data better by being careful, following rules, and thinking about what we're doing.
